modules/ROOT/pages/indexes-for-vector-search.adoc

```diff
@@ -21,10 +21,11 @@ An embedding is a numerical representation of a data object, such as text, image
 
 For example, each word or token in a text is typically represented as high-dimensional vector where each dimension represents a certain aspect of the word’s meaning.
 Words that are semantically similar or related are often represented by vectors that are closer to each other in this vector space.
-This allows for mathematical operations like addition and subtraction to carry semantic meaning; for example, the vector representation of "king" minus "man" plus "woman" might be close to the vector representation of "queen." In other words, dimensional vector embeddings can be said to be a numerical representation of a particular data object, capturing its semantic meaning.
-The embedding for a particular data object can be generated by for example the https://cloud.google.com/vertex-ai[Vertex AI] or https://openai.com/[OpenAI] embedding generators, which can produce dimensional vector embeddings with lengths 768, 1024, 1536, etc.
-These dimensional vector embeddings are stored as `LIST<FLOAT>` properties in a Neo4j vector index.
-(It can be used to index nodes by `LIST<FLOAT>` properties valid to the index.)
+This allows for mathematical operations like addition and subtraction to carry semantic meaning; for example, the vector representation of "king" minus "man" plus "woman" might be close to the vector representation of "queen." In other words, vector embeddings can be said to be a numerical representation of a particular data object, capturing its semantic meaning.
+
+The embedding for a particular data object can be generated by for example the https://cloud.google.com/vertex-ai[Vertex AI] or https://openai.com/[OpenAI] embedding generators, which can produce vector embeddings with dimensions 768, 1024, 1536, etc.
+These vector embeddings are stored as `LIST<FLOAT>` properties on a node, where each dimensional component of the vector, is an element in the `LIST`.
+A Neo4j vector index can be used to index nodes by `LIST<FLOAT>` properties valid to the index.
```

Comment:
This last sentence may want a slight reword. It feels like there are too many `index`es